horovod/tensorflow/__init__.py

```diff
@@ -97,6 +97,17 @@ def broadcast_global_variables(root_rank):
     return tf.group(*[tf.assign(var, broadcast(var, root_rank))
                       for var in tf.global_variables()])
 
+def bcast(root_rank, variables):
```

Comment:
Can you rename this to `broadcast_variables`?

Reply:
sure. rename to ```broadcast_variable```